gulpfile.js

```diff
@@ -490,11 +490,11 @@ gulp.task('release', ['apps', 'clean-release'], function (done) {
     }
 
     if (platforms.indexOf('win32') !== -1) {
-        merged.add(releaseWin('win32'));
+        return release_win('win32');
```

Comment:
Without `return`, if you put a `return` you can't create a `--win32` and `--win64` version at the same time.

Reply:
Yeah, I wasn't sure about that and on my work laptop I can't run npm (behind an old fashioned proxyserver). Without the `return` does it finishes the task in a neat way? I expect that gulp will report that the `release` task is finished while the build is still running.

Reply:
Yes, without the return the task finishes without problem. All the streams execute at the same time, but the win32 task and the win64 task execute one after the other, and until not finished, the code does not arrive to the `merge`, that does what it is intended to do.